rotkehlchen/data_import/importers/binance.py

```diff
@@ -273,8 +272,11 @@ def process_trades(
                 cur_batch.append(rows_grouped_by_fee[True].pop())
             grouped_trade_rows.append(cur_batch)
 
-        # Creating trades structures based on grouped rows data
-        raw_trades: list[Trade] = []
+        # Sometimes we can get absolutely identical swap events
+        # (including timestamp) but the database allows us to add only one of them.
+        # So we combine these swap events into a huge group of swap events
+        # First step: group swap events
```

Comment:
```suggestion
        # First step: create and group swap events
```
we're not just grouping them here, this is also where they get created.